README.md

```diff
@@ -186,14 +186,21 @@ The `pathfinder` node can be run in the provided Docker image.
 
 ```bash
 docker run \
-  -e RUST_LOG=info \
   -p 9545:9545 \
-  eqlabs/pathfinder \
-    --http-rpc="0.0.0.0:9545" \
-    --ethereum.url="https://goerli.infura.io/v3/<project-id>" \
-    --ethereum.password="<password>"
+  -e RUST_LOG=info \
+  -e PATHFINDER_ETHEREUM_API_URL="https://goerli.infura.io/v3/<project-id>" \
+  eqlabs/pathfinder
 ```
 
+The following environment variables can be passed to the container:
+
+| Name                               | Description                                                  | Default value | Required |
+| ---------------------------------- | ------------------------------------------------------------ | ------------- | -------- |
+| PATHFINDER_ETHEREUM_API_URL        | Ethereum (archive) node JSON-RPC endpoint URL                |               | yes      |
+| PATHFINDER_ETHEREUM_API_USER_AGENT | User-Agent header value to use with Ethereum node API        |               | no       |
```

Comment:
I presume this means this can only be merged after #223 then?

Reply:
Yes.